core/primitives-core/src/version.rs

```diff
@@ -219,24 +219,26 @@ impl ProtocolFeature {
             ProtocolFeature::DecreaseFunctionCallBaseCost => 66,
             ProtocolFeature::YieldExecution => 67,
 
+            // Congestion control should be enabled BEFORE stateless validation, so it has a lower version.
+            ProtocolFeature::CongestionControl => 80,
+
+            // Stateless validation features.
+            ProtocolFeature::StatelessValidationV0
+            | ProtocolFeature::LowerValidatorKickoutPercentForDebugging
+            | ProtocolFeature::SingleShardTracking
+            | ProtocolFeature::StateWitnessSizeLimit
+            | ProtocolFeature::PerReceiptHardStorageProofLimit
+            | ProtocolFeature::PartialEncodedStateWitness
+            | ProtocolFeature::WitnessTransactionLimits
+            | ProtocolFeature::OutgoingReceiptsSizeLimit
+            | ProtocolFeature::NoChunkOnlyProducers
+            | ProtocolFeature::ChangePartialWitnessDataPartsRequired
+            | ProtocolFeature::BiggerCombinedTransactionLimit => 81,
+
             // This protocol version is reserved for use in resharding tests. An extra resharding
             // is simulated on top of the latest shard layout in production. Note that later
             // protocol versions will still have the production layout.
-            ProtocolFeature::SimpleNightshadeTestonly => 79,
-
-            // StatelessNet features
-            ProtocolFeature::CongestionControl => 80,
-            ProtocolFeature::StatelessValidationV0
-            | ProtocolFeature::LowerValidatorKickoutPercentForDebugging => 81,
-            ProtocolFeature::SingleShardTracking => 82,
-            ProtocolFeature::StateWitnessSizeLimit => 83,
-            ProtocolFeature::PerReceiptHardStorageProofLimit => 85,
-            ProtocolFeature::PartialEncodedStateWitness => 86,
-            ProtocolFeature::WitnessTransactionLimits
-            | ProtocolFeature::OutgoingReceiptsSizeLimit => 87,
-            ProtocolFeature::NoChunkOnlyProducers => 88,
-            ProtocolFeature::ChangePartialWitnessDataPartsRequired => 89,
-            ProtocolFeature::BiggerCombinedTransactionLimit => 90,
+            ProtocolFeature::SimpleNightshadeTestonly => 100,
```

Comment:
This might not work for the test in resharding... Let me check

Reply:
Seems like we are anyway ignoring all resharding tests. Might be fine to just ignore test_latest_protocol_missing_chunks_low_missing_prob etc. too in case it fails.

Reply:
Is it a Nayduck test?

Reply:
They did not fail, so I guess the new version is ok.

Reply:
Generally speaking is deemed to be buried until we start working on resharsing again. Most of the resharding tests are disabled because of lack of integration with cc and sv anyway. Feel free to add `ignore` to any other tests that may fail because of this change.